tests/integration/test_containerd_integration.py

```diff
@@ -43,42 +43,66 @@ async def test_status_messages(ops_test):
 async def test_upgrade_containerd_action(ops_test):
     """Test running upgrade-containerd action."""
     unit = ops_test.model.applications["containerd"].units[0]
-    action = await unit.run_action("upgrade-containerd")
```

Comment:
even though the action is deprecated, do we want to make upgrade-containerd untested?